static/js/publisher/metrics/graphs/activeDevices.js

```diff
@@ -86,14 +91,64 @@ const positionLabels = function(el, activeDevices) {
   });
 };
 
-export default function activeDevices(days, activeDevices) {
+export default function activeDevices(days, activeDevices, type) {
   const el = document.getElementById('active_devices');
 
+  activeDevices.sort((a, b) => {
+    const a1 = a.slice(1);
```

Comment:
Why are first items removed from these arrays?
I guess the structure of the data explains it, but it's not clear from the code. Would be useful to leave some comment in code.